package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "zipkin-api",
+  "version": "0.0.0",
+  "description": "Zipkin API ",
+  "main": "validate.js",
+  "scripts": {
+    "test": "node validate.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openzipkin/zipkin-api.git"
+  },
+  "keywords": [
+    "zipkin",
+    "api"
+  ],
+  "author": "",
+  "license": "ISC",
```

Comment:
Why is this ISC license? All of zipkin things are ASL. I'm worried we will have the burden of explaining this.